src/featurizers/mitie_featurizer.py

```diff
@@ -3,28 +3,36 @@
 import urllib2
 import os
 import httplib
-
+import multiprocessing
 
 class MITIEFeaturizer(object):
 
     def __init__(self, fe_file):
         if not os.path.isfile(fe_file):
-            self.download_fe_file(fe_file)
+            self.download(fe_file)
+            return  # need a cleaner solution for this
         self.feature_extractor = total_word_feature_extractor(fe_file)
         self.ndim = self.feature_extractor.num_dimensions
 
     def download_fe_file(self, fe_file):
-        chunk_size = 4096
+        print("DOWNLOADING MITIE FILE")
+        chunk_size = 30000000
         _fe_file = urllib2.urlopen("https://s3-eu-west-1.amazonaws.com/mitie/total_word_feature_extractor.dat")
```

Comment:
we could probably add this URL transparently when creating the RASANLUConfiguration, so that you could refer to it as config.mitie_word_feature_extractor_url or something similar